lib/ansible/modules/web_infrastructure/jenkins_plugin.py

```diff
@@ -172,13 +172,14 @@
 #
 # my_jenkins_params:
 #   url_username: admin
+#   url: https://localhost:8433
```

Comment:
Just thinking that it would be better to keep the example with `http://` only because that's how it usually runs when you manage the Jenkins server directly. So the `validate_certs` is not required here. Moving the `url` param here is good, although not required if Jenkins runs on the default port (8080) as that's the default value of the parameter.

Reply:
I was thinking that this is the most common scenario (SSL port with an ad-hoc certificate), which is why it is more useful to document it this way. Besides, the idea is to show the names of the parameters and how to use them, the actual values shouldn't really matter. Once the reader knows how to pass url, they can easily modify it to suit their needs. The same is the case with the cert option, i.e., assuming most don't bother getting a CA cert, it shows one how to disable validation, but if one is running with a valid cert, then it is matter of removing the line, or changing `no` to `yes`.

Reply:
But, if you feel strongly about it, I will change it back.

Reply:
If you accessing service via localhost, there is no need to use SSL.

Reply:
I followed a best practice and set `JENKINS_PORT` to `-1` to disable non-SSL port. Anyway, this is not a big deal, I can revert. Only the change to add the `url` line will remain, and since it would work for the default case without having to specify it, there is nothing left to merge here :)

Reply:
Yeah. But at least you tried and I had the chance to chat to you ;o)

Reply:
Cool, I just closed this PR. I hope to collaborate more in the future.